### ISSUE TITLE
Submit Event Anonymously

### ISSUE THREAD

**Issue by mpotgieter** [MISP/MISP#1954]
It would be useful to have the functionality to submit an event with attributes anonymously. In our instance of MISP we have several organisations and at times individuals would like to disassociate the organisation name from the event submitted. Maybe there could be an "anonymous" organisation that the events can be attached to, but because a contributor plays a role in various workflows this may introduce issues.
There may be other ways to accomplish this such as creating a web form that submits via the MISP API, but that introduces other issues, such as the requirement that someone review and publish the event. 
I am open to suggestions on how to handle this.

**Comment by iglocska** [comment]
We have something called "delegations" to allow for a way of handling similar scenarios. Basically the idea is that a trusted 3rd party takes over an event and publishes under their name (this could very easily be an anonymous organisation on your instance). 

All you need to do is create an event set to "your organisation only" and delegate the event to a 3rd party organisation. That organisation will get notified via e-mail and can accept the delegation request. The event is then deleted and a new duplicate is created in the name of the target organisation, at which point they can set the distribution level and release the event as their own.

Is this somethig that would work for you?

**Comment by mpotgieter** [comment]
Tried to test this out but do not know where to do the delegation, even after assigning the "delegate" permission and looks like the docs have not been updated on this feature as I could not find it in the guide. If you can guide me to where I can delegate an event, I will test and see if it meets my requirements. Thanks

**Comment by iglocska** [comment]
First of all make sure that delegations are enabled on your instance:

(Administration -> Server settings -> MISP tab -> delegation setting -> true)

After that, create an event with the distribution set to "my organisation only" and the delegation button should become available right below the publish buttons on the left panel. 

Let me know if you have any other questions!

**Comment by adulau** [comment]
@mpotgieter Did the delegation feature supported your requirements? We are interested in any feedback.

**Comment by mpotgieter** [comment]
We tested the delegation feature and the delegation request is only delegated once it is accepted, and during the time that it is under request for delegation, there is a possibility that the request will be visible by others. Of course limiting access to "this Organisation" would limit who would be able to see. I also looked at the event history and there is no record of who created the original event. 
So it looks like this may work. Thanks for the quick response and assistance.
